client/src/pages/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import { Box, Typography, Paper } from '@material-ui/core';
+import ForgotPasswordForm from './ForgotPasswordForm/ForgotPasswordForm';
+import { forgotPassword } from '../../helpers/APICalls/forgotPassword';
+import useStyles from './useStyles';
+
+export default function Reset(): JSX.Element {
+  const [hasSubmitted, setHasSubmitted] = useState<boolean>(false);
+  const [error, setError] = useState<string>();
+  const classes = useStyles();
+
+  const handleSubmit = (
+    { email } : { email: string }
+  ) => {
+    forgotPassword(email)
+      .then((res) => {
+        if (res.error) {
+          setError(res.error.message);
+        }
+      });
+    setHasSubmitted(prev => !prev);
+  };
+
+  return (
+    <Box className={classes.container}>
```

Comment:
The Box component has flexbox built into it, you can add props to it rather than using a class here.

Reply:
Wouldn't it be better to have it in the useStyles file instead of being clumped up like this?
```
<Box className={classes.container} display="flex" justifyContent="center" alignItems="center" flexDirection="column">
```

Reply:
Good point! I guess when you have other styles in addition to flexbox, you can use classes but if it's only for flexbox then maybe add props to the component :)